pkg/api/pluginproxy/ds_proxy.go

```diff
@@ -311,8 +314,9 @@ func addOAuthPassThruAuth(c *models.ReqContext, req *http.Request) {
 		return
 	}
 
-	provider := authInfoQuery.Result.AuthModule
-	connect, ok := social.SocialMap[strings.TrimPrefix(provider, "oauth_")] // The socialMap keys don't have "oauth_" prefix, but everywhere else in the system does
+	// The socialMap keys don't have "oauth_" prefix, but everywhere else in the system does
+	provider := strings.TrimPrefix(authInfoQuery.Result.AuthModule, "oauth_")
+	connect, ok := social.SocialMap[provider]
```

Comment:
Maybe can create a function for this in social package that encapsulates the handling of the "oauth_" prefix and getting a provider/connect from SocialMap?

Reply:
Done